src/Tracking.php

```diff
@@ -100,7 +100,7 @@ public static function maybe_init() {
 			}
 
 			// AddToCart - ajax.
-			if ( 'yes' === get_option( 'woocommerce_enable_ajax_add_to_cart' ) && 'yes' !== get_option( 'woocommerce_cart_redirect_after_add' ) ) {
+			if ( 'yes' !== get_option( 'woocommerce_cart_redirect_after_add' ) ) {
```

Comment:
I'd like to understand why this condition was there in the first place. I checked the `ajax_tracking_snippet` and can see that it attaches a JS function to the `added_to_cart` event. 

Do we need these conditions if this event is triggered without these options? Can you please provide some context here.

Reply:
Thank you so much, @message-dimke, for the time to explain the context of the condition. I really appreciate it.